domain/resource.go

```diff
@@ -13,10 +13,11 @@ type Resource struct {
 	Details      map[string]interface{} `json:"details" yaml:"details"`
 	Labels       map[string]string      `json:"labels,omitempty" yaml:"labels,omitempty"`
 	CreatedAt    time.Time              `json:"created_at,omitempty" yaml:"created_at,omitempty"`
-	UpdatedAt    time.Time              `json:"updated_at,omitempty" yaml:"updated_at,omitempty"`
+	UpdatedAt    time.Time              `json:"updated_at,omitempty" yaml:"updatesd_at,omitempty"`
```

Comment:
can you correct this
```suggestion
	UpdatedAt    time.Time              `json:"updated_at,omitempty" yaml:"updated_at,omitempty"`
```